spec/components/schemas/Payments/PaymentRequest.yaml

```
@@ -144,6 +144,14 @@ properties:
                 type: string
                 description: The cardholder's email address.
                 example: "john.doe@example.com"
+          installments:
+            type: object
+            description: Details about the installments.
+            properties:
+              count:
+                type: string
+                description: The number of payments to be made per year.
```

I'm not sure this is something associated with a yearly cycle.
It's the number of payments that a single auth will be broken up into.

---

```suggestion
                description: The number of payments to be made in the installment plan.
```
Does that work better?

---

Yip, LGTM!